.circleci/config.yml

```diff
@@ -26,6 +26,7 @@ jobs:
           name: Install python dependencies
           # FIXME: For now, we fix setuptools due to https://github.com/etalab/data.gouv.fr/issues/1041
           command: |
+            python --version
```

Comment:
Why do you need to print python version here?

Reply:
It was for debug (I had problems with Python 3.7 in CI/production since we are on 3.9 in local dev)